rerun_cpp/README.md

```diff
@@ -1,10 +1,61 @@
 # Rerun C++ SDK
 
-Read our [getting-started guide](https://www.rerun.io/docs/getting-started/cpp) for information on how to use the Rerun C++ SDK.
+The Rerun C++ SDK allows logging data to Rerun directly from C++.
 
-## Development
-### Requirements
-Run `scripts/setup.sh`.
+Read the [getting started guide](https://www.rerun.io/docs/getting-started/cpp) for information on how to use the Rerun C++ SDK.
 
-### Test it
-`just cpp-test`
+## Build & Distribution
+
+### Overview
+
+To avoid compatibility issues across different platforms, compiler versions and C++ standard library versions
+the C++ SDK is expected to be built from source.
+
+From a build system perspective, the SDK consists of three dependencies:
+
+* [SDK source](src/)
+  * this includes **both** source and header files!
+* [rerun_c](../crates/rerun_c/) static libraries
```

Comment:
If this is included in the external zip maybe link explicitly to github?

Reply:
yeah good point I should make all these links more robust!